src/schema.rs

```diff
@@ -236,219 +217,303 @@ fn renomination() -> TransitionSchema {
+            root_id: SchemaId::from_str(BECH32_SCHEMA_ID_ROOT)
+                .expect("Broken root schema ID for RGB20 sub-schema"),
+            type_system: none!(),
+            genesis: GenesisSchema {
                 metadata: type_map! {
-                    FieldType::BurnedSupply => Once,
-                    // Normally issuer should aggregate burned assets into a
-                    // single UTXO; however if burn happens as a result of
-                    // mistake this will be impossible, so we allow to have
-                    // multiple burned UTXOs as a part of a single operation
-                    FieldType::BurnUtxo => OnceOrMore,
+                    FieldType::Ticker => Once,
+                    FieldType::Name => Once,
+                    FieldType::Precision => Once,
+                    FieldType::Timestamp => Once,
                     // We need this field in order to be able to verify pedersen
                     // commitments
-                    FieldType::IssuedSupply => Once,
-                    FieldType::HistoryProofFormat => Once,
-                    FieldType::HistoryProof => NoneOrMore
-                },
-                closes: type_map! {
-                    OwnedRightType::BurnReplace => Once
+                    FieldType::IssuedSupply => Once
                 },
                 owned_rights: type_map! {
-                    OwnedRightType::BurnReplace => NoneOrOnce,
-                    OwnedRightType::Assets => OnceOrMore
-                },
-                public_rights: none!()
-            },
-            TransitionType::Renomination => renomination(),
-            // Allows split of rights if they were occasionally allocated to the
-            // same UTXO, for instance both assets and issuance right. Without
-            // this type of transition either assets or inflation rights will be
-            // lost.
-            TransitionType::RightsSplit => TransitionSchema {
-                metadata: type_map! {},
-                closes: type_map! {
                     OwnedRightType::Inflation => NoneOrMore,
-                    OwnedRightType::Assets => NoneOrMore,
-                    OwnedRightType::OpenEpoch => NoneOrOnce,
-                    OwnedRightType::BurnReplace => NoneOrMore,
-                    OwnedRightType::Renomination => NoneOrOnce
+                    OwnedRightType::Assets => NoneOrMore
                 },
-                owned_rights: type_map! {
-                    OwnedRightType::Inflation => NoneOrMore,
-                    OwnedRightType::Assets => NoneOrMore,
-                    OwnedRightType::OpenEpoch => NoneOrOnce,
-                    OwnedRightType::BurnReplace => NoneOrMore,
-                    OwnedRightType::Renomination => NoneOrOnce
+                public_rights: none!(),
+            },
+            extensions: none!(),
+            transitions: type_map! {
+                TransitionType::Issue => issue(),
```

Comment:
not sure calling `issue()` here is correct, since the method allows a `OwnedRightType::OpenEpoch` which is not defined in the rgb20_inflationary subschema `owned_right_types`